pkg/hooks/audit_log_file.go

```diff
@@ -0,0 +1,45 @@
+package hooks
+
+import "github.com/square/p2/pkg/logging"
+
+// Returns a FileAuditLogger using the given logger
+func NewFileAuditLogger(logger *logging.Logger) *FileAuditLogger {
```

Comment:
Do you prefer FileAuditLogger or LoggerAuditLogger? Maybe there's a non-derpy name for this thing that somehow implies that it logs auditfully to a logger.